### ISSUE TITLE
PieChart, adding an imageView beside values 

### ISSUE THREAD

**Issue by maryamfekri** [danielgindi/Charts#2318]
can you please help me how to add an imageView beside the position of the label like below image :
![nehew](https://cloud.githubusercontent.com/assets/13133764/24608137/6e704dde-188a-11e7-9611-39adb6d70900.png)

**Comment by liuxuan30** [comment]
The first question seems too generic and the answer you should already know. The real issue should be your second qestion, It's better can can explain with details for
>but it does not draw images on a good position regard to the related value label. it has an iconOffset with x and y but it won't fully support for all different values.

**Comment by maryamfekri** [comment]
at first I didn't know about the feature of having icon in your new version, but then I realize that there is a parameter for DataEntry that receive an NSUIIMage. but it will place the image in side the pie chart. 
But what i need for my project is to have the image beside the labels (see the image above -> 5%).

I found that the icon has an offset property, which will receive x and y for moving the image, but It does not handle the overlaying of images. I mean if use this icon you provide in your new version and moving it to outside the pie chart, it will overlay with other images and labels. 

can you please fix this issue to handle this?  or maybe you need a Bool to determine image should be outside the piechart or inside the piechart.

**Comment by maryamfekri** [comment]
I have also posted a question in stackOverFlow which you can track [here](http://stackoverflow.com/questions/42113512/add-an-image-for-each-slice-in-ios-chart-piechart)

**Comment by maryamfekri** [comment]
I developed in my project and also committed here as a merge request. 
the purpose is to add an image beside the value in pie chart.

**Comment by sashalondon** [comment]
I cannot find any reference in the wiki on how to do this?

**Comment by maryamfekri** [comment]
@sashalondon  yes , still not merged with the origin repo, but I have created the pull request which is waiting to be merge. although you can find my commits in my [pull request](https://github.com/danielgindi/Charts/pull/2483)
